tee-worker/litentry/core/stf-task/receiver/src/lib.rs

```diff
@@ -179,11 +180,20 @@ where
 			.recv()
 			.map_err(|e| Error::OtherError(format!("receiver error:{:?}", e)))?;
 
+		let start_time = std::time::Instant::now();
+
 		match &req {
 			RequestType::IdentityVerification(req) =>
 				IdentityVerificationHandler { req: req.clone(), context: context.clone() }.start(),
 			RequestType::AssertionVerification(req) =>
 				AssertionHandler { req: req.clone(), context: context.clone() }.start(),
 		}
+
+		if let Err(e) = context.ocall_api.update_metric(EnclaveMetric::StfTaskExecutionTime(
+			Box::new(req),
```

Comment:
Hmm -- will this print out anything regarding the request? If so we should kind of avoid it

Reply:
Ah I see the censoring below, so this is resolved